test-files/cmd/slow/DeriveChangesetCmdTest/changeset-1.osc

```diff
@@ -9,6 +9,7 @@
             <nd ref="-2"/>
             <tag k="highway" v="road"/>
             <tag k="note" v="1"/>
+            <tag k="error:circular" v="15"/>
         </way>
```

Comment:
A side effect of fixing the default CE option handling is seeing error:circular in some of the test output where it wasn't before.  This is ok, IMO, as it can be removed if needed by specifying writer.include.circular.error.tags=false.